### DISCUSSION TITLE
Importing notebooks from OneNote and getting Wacom buttons to work

### DISCUSSION THREAD

**Discussion by DeltaV39** [flxzt/rnote#977] [Q&A]
Hello, I've recently discovered Rnote and am trying to switch from OneNote. Already I'm seeing how much more optimised it is. However, I'm having a bit of trouble figuring a few things out so I have a few questions which may or may not have already been asked.

1. Is there any way to transfer entire notebooks fron OneNote via import/export (i.e. not just exporting individual pages and sections as .pdfs and importing them for annotation)? Also, is it possible to import pages and keep the existing ink strokes editable, without having to bake them into a pdf when exporting?
2. I have a Wacom tablet and the Wacom driver comes with an app that seems to override the pen and tablet button mappings, so that I can't use the ones in Rnote. What can I do about this?

**Comment by flxzt** [comment]
Hi, let me try to answer your questions:

1.: no, currently not. There is an open issue #744, progress towards a onenote import will be tracked there.
2.: I am not completely certain how the situation on windows is regarding stylus support with Gtk (the UI-Toolkit used by Rnote). Are you able to set the mapping to "default" or "app" or something similar?

**Reply by DeltaV39** [reply]
Thanks for your reply! I am able to set the mapping to "default" in the configuration program, however this is Wacom's default mapping, which still overwrites the "Button 1", "Button 2", etc. within RNote. Here are some screenshots to show what I mean:
![image](https://github.com/flxzt/rnote/assets/48811086/8d13a372-c944-4dfd-9824-a41d0899ca6c)
![image](https://github.com/flxzt/rnote/assets/48811086/a3df274e-715f-43ad-966a-5909db4b3d2d)
Should I assume the only way to fix this is to switch to Linux? I would like to, but don't currently have access to a USB drive to install a distro on. 

I'm looking forward to the OneNote import though. I didn't know that the web version of OneNote can't open documents from the desktop version, which is hilariously janky. According to https://github.com/flxzt/rnote/discussions/733#discussioncomment-6415922, variable width strokes don't get exported in the method described by tobiaskramer. Does this mean that they will be converted to constant-width strokes or just not appear?
